app/models/encrypted_field.rb

```diff
@@ -1,6 +1,7 @@
 class EncryptedField
   KEY = ActiveSupport::KeyGenerator.new(
-    Rails.application.secret_key_base
+    Rails.application.secret_key_base,
+    hash_digest_class: OpenSSL::Digest::SHA1
```

Comment:
If we're explicitly saying to use SHA1 rather than SHA256, why do we need to do a database migration to `User`. I assumed the `User` updates were to re-encrypt the files as default.

Reply:
user model change missing from the previous commits to specify new encrypted field

Reply:
OK, so this is to ensure the existing EncryptedFields work as before, but User model has changed to use built in support.